### PR TITLE
Fix No module error in GridBasedSweepCPP and ClosedLoopRRTStart

### LINKED ISSUE #515: 'No module named' error for many examples.
```
File "grid_based_sweep_coverage_path_planner.py", line 20, in <module>
    from Mapping.grid_map_lib.grid_map_lib import GridMap
ModuleNotFoundError: No module named 'Mapping' 
```

I was trying to run a grid_based_sweep_coverage_path_planning and encountered ModuleNotFoundError on custom library(Module) that you created. I noticed that you recently made commit(#456) on few examples that related to sys.path.append.
I could run without error after adding sys.path.append to main code(grid_based_sweep_coverage_path_planning.py) from __init__.py

```
import math
from enum import IntEnum

import numpy as np
from scipy.spatial.transform import Rotation as Rot
import matplotlib.pyplot as plt
import os
import sys

GRID_MAP_LIB = os.path.dirname(os.path.abspath(__file__)) + \
               "/../../Mapping/"

sys.path.append(GRID_MAP_LIB)

try :
    from grid_map_lib.grid_map_lib import GridMap
except ImportError:
    raise
```

I also notice that some of the examples still doesn't have __init__.py file. 
Is there any reason you separate sys.path.append part to __init__.py?
and i don't know if the code is wrong or i am doing something wrong. 

frenet_optimal_trajectory.py has sys.path.append inside its own code works fine.
grid_based_sweep_coverage_path_planner.py, closed_loop_rrt_star_car.py has sys.path.append inside __init__.py code which
gives ModuleNotFoundError.  

**Desktop (please complete the following information):**
 - Python version : 3.8.10
 - Each library version

### PR DESCRIPTION
**Pull request by benthebear93**
#### Reference issue
fix #515

#### What does this implement/fix?
changed import Module path.
all the other codes has sys.path.append code in the main python code except
GridBasedSweepCPP and ClosedLoopRRTStart has in __init__.py which makes "No module error"

### CODE REVIEW

File: PathPlanning/GridBasedSweepCPP/grid_based_sweep_coverage_path_planner.py [hunk 1]
```diff
@@ -301,7 +311,7 @@ def main():  # pragma: no cover
 
     ox = [0.0, 20.0, 50.0, 100.0, 130.0, 40.0, 0.0]
     oy = [0.0, -20.0, 0.0, 30.0, 60.0, 80.0, 0.0]
-    resolution = 5.0
```
Comment (AtsushiSakai) [inline PathPlanning/GridBasedSweepCPP/grid_based_sweep_coverage_path_planner.py:304]: Why do you want to change this? It looks unrelated to import problem.

File: PathPlanning/GridBasedSweepCPP/grid_based_sweep_coverage_path_planner.py [hunk 2]
```diff
@@ -249,13 +258,14 @@ def planning(ox, oy, resolution,
...
         ox, oy)
+    print("sweep_vec", sweep_vec, sweep_start_position)
 
     rox, roy = convert_grid_coordinate(ox, oy, sweep_vec,
                                        sweep_start_position)
 
     grid_map, x_inds_goal_y, goal_y = setup_grid_map(rox, roy, resolution,
                                                      sweeping_direction)
-
+    print(x_inds_goal_y, goal_y)
```
Comment (AtsushiSakai) [inline PathPlanning/GridBasedSweepCPP/grid_based_sweep_coverage_path_planner.py:268]: Why do you want to change this? It looks unrelated to import problem.

File: PathPlanning/GridBasedSweepCPP/grid_based_sweep_coverage_path_planner.py [hunk 3]
```diff
@@ -249,13 +258,14 @@ def planning(ox, oy, resolution,
              ):
     sweep_vec, sweep_start_position = find_sweep_direction_and_start_position(
         ox, oy)
+    print("sweep_vec", sweep_vec, sweep_start_position)
```
Comment (AtsushiSakai) [inline PathPlanning/GridBasedSweepCPP/grid_based_sweep_coverage_path_planner.py:261]: Why do you want to change this? It looks unrelated to import problem.

**Review by AtsushiSakai — COMMENTED** [review]
Thank you for your PR. I have some questions. PTAL.

File: PathPlanning/GridBasedSweepCPP/grid_based_sweep_coverage_path_planner.py (hunk 3 above)
Comment (benthebear93) [reply PathPlanning/GridBasedSweepCPP/grid_based_sweep_coverage_path_planner.py]: This was my mistake. I forgot to remove some line that i tested. Sorry for inconvenience.

**Review by AtsushiSakai — APPROVED** [review]

**Comment by AtsushiSakai** [comment]
I will merge it. Thanks @benthebear93
